code/modules/antagonists/gang/gang_leader.dm

```diff
@@ -0,0 +1,125 @@
+/datum/antagonist/gang_leader
```

Comment:
i feel like this should be a subtype of the gang member datum?

Reply:
Gang leader datums being a subtype of gang member datums would introduce the issue of gang leaders being treated as subordinate antagonists; already I feel that the two datums are sufficiently different to merit distinct types.